demo/imdb/imdb_queries.py

```diff
@@ -158,6 +158,22 @@
                      ['Judi Dench', '84.000000']]
 )
 
+actors_over_85_index_scan = QueryInfo(
+    query="""MATCH (a:actor)
+             WHERE a.age > 85
+             RETURN a.name, a.age""",
```

Comment:
MATCH queries do not guarantee order, but the index scan will return values in their stored order (which for this, is increasing age). As `GRAPH.EXPLAIN` won't currently describe index scans, this is valuable right now for making sure the optimization is being used.

Reply:
We can apply optimisations when the client issued an explain query,
awhile back one of our optimisations required actual data ingestion, but this optimisation was removed.